chapters/sampling-randomization-power.tex

```diff
@@ -1,391 +1,443 @@
+and define it as the difference between the true treatment effect
+and the estimated effect due to placing units in groups.
+The intuition for both measures is that from any group,
+you can find some possible subsets that have higher-than-average values of some measure;
+similarly, you can find some that have lower-than-average values.
+Your sample or randomization will inevitably fall in one of these categories,
+and we need to assess the likelihood and magnitude of this occurence.\sidenote{
+  \url{https://davegiles.blogspot.com/2019/04/what-is-permutation-test.html}}
+Power calculation and randomization inference are the two key tools to doing so.
+
+\subsection{Power calculations}
+
+Power calculations report the likelihood that your experimental design
+will be able to detect the treatment effects you are interested in.
+This measure of \textbf{power} can be described in various different ways,
+each of which has different practical uses.\sidenote{
+  \url{http://www.stat.columbia.edu/~gelman/stuff_for_blog/chap20.pdf}}
+The purpose of power calculations is to identify where the strengths and weaknesses
+of your design are located, so you know the relative tradeoffs you will face
+by changing your randomization scheme for the final design.
+They also allow realistic interpretations of evidence:
+results of low-power studies can be very interesting,
+but they have a correspondingly higher likelihood
+of reporting false positive results.
+
+The classic definition of power
+is the likelihood that a design detects a significant treatment effect,
+given that there is a non-zero true effect in reality.
+This definition is useful retrospectively,
+but it can also be re-interpreted to help in experimental design.
+There are two common and useful practical applications
+of that definition that give actionable, quantitative results.
+The \textbf{minimum detectable effect (MDE)}\sidenote{
+  \url{https://dimewiki.worldbank.org/wiki/Minimum_Detectable_Effect}}
+is the smallest true effect that a given research design can detect.
+This is useful as a check on whether a study is worthwhile.
+If, in your field, a ``large'' effect is just a few percentage points
+or a fraction of a standard deviation,
+then it is nonsensical to run a study whose MDE is much larger than that.
+This is because, given the sample size and variation in the population,
+the effect needs to be much larger to possibly be statistically detected,
+so such a study would not be able to say anything about the effect size that is practically relevant.
+Conversely, the \textbf{minimum sample size} pre-specifies expected effect sizes
+and tells you how large a study's sample would need to be to detect that effect,
+which can tell you what resources you would need to avoid that exact problem.
+
+Stata has some commands that can calculate power analytically for
 very simple designs -- \texttt{power} and \texttt{clustersampsi} --
 but they will not answer most of the practical questions
```

Comment:
you can do a fair amount with power. we should be more explicit about what it can't do, and more specific on the advantages of using simulations instead.

Reply:
issue #339